presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestExplainVerification.java

```diff
@@ -74,7 +74,7 @@ public void testDetailsMismatch()
         assertTrue(event.isPresent());
 
         // Explain verification do not fail in case of plan changes.
-        assertEvent(event.get(), SUCCEEDED);
+        assertEvent(event.get(), FAILED);
```

Comment:
The comment above says that Explain verification do not fail in case of plan changes. Can you comment why it should be failed?